psi4/src/psi4/libcubeprop/csg.cc

```diff
@@ -649,4 +664,36 @@ void CubicScalarGrid::compute_ELF(std::shared_ptr<Matrix> D, const std::string&
     write_gen_file(v, name, type);
     delete[] v;
 }
+std::pair<double, double> CubicScalarGrid::compute_isocontour_range(double* v2, double exponent) {
```

Comment:
Not sure where this is called from, but would it make more sense to pass in a managed object so that it can be called Python side?

Reply:
Let me think about this one.

Reply:
This function is used internally and I think it would be difficult to interface it to Python. What we really need is a Cubefile object with py-side interface (for another PR). That would allow us to find the isocontour values directly from Python and play with different values without having to rerun Psi4. However, I think the current default density threshold (85%) is pretty good. BTW, the reason I am not doing all of this in Python is because by the time we write all the data to a cube file it takes about 2 seconds/cube to re-read the data even with  np.genfromtxt.